source/jormungandr/jormungandr/interfaces/v1/Uri.py

```diff
@@ -100,6 +106,10 @@ def get(self, region=None, lon=None, lat=None, uri=None, id=None):
 
         args = self.parsers["get"].parse_args()
 
+        # for retrocompatibility purpose
+        for forbid_id in args.get('forbidden_ids[]', []):
```

Comment:
nop seems that only one param can write in a `dest`, so the old param was not working